cryptus-app/lib/update_nfts_for_user.ts

```diff
@@ -28,24 +34,64 @@ export default async function updateNftsForUser(
 
     nfts_raw = data.assets;
   } catch (err) {
-    console.error(err);
-    console.log("response = ", res);
+    // console.error(err);
+    // console.log("response = ", res);
     return null;
   }
 
   // console.log("traits = ", nfts_raw[0]);
-
   const nft_clean = nfts_raw.map((nft) => {
-    const properties = JSON.stringify(
-      nft.traits.map((trait) => {
-        return {
+    // Sort properties here
+    const properties: traits[] = [];
+    if (!(nft.traits === undefined) && !(nft.traits.length == 0)) {
+      const first_trait: traits = {
+        name: nft.traits[0].trait_type,
```

Comment:
use this instead:
<img width="337" alt="image" src="https://user-images.githubusercontent.com/32850285/153914402-a5479ed6-a25d-4063-a7ee-8e1e145fae40.png">